### PR TITLE
Update source files for 2019 data refresh

### PR DESCRIPTION
**Pull request by ajmccarthy**
Update to liquibase files to pull data from new 2019 source files

### CODE REVIEW

File: liquibase_files/changelog1DailyTables.sql [hunk 1]
```diff
@@ -146,7 +145,9 @@ CREATE TABLE SEDMAP.DAILY_YEARS
   "2014"                VARCHAR2(255 BYTE),
   "2015"                VARCHAR2(255 BYTE),
   "2016"                VARCHAR2(255 BYTE),
-  "2017"                VARCHAR2(255 BYTE)
+  "2017"                VARCHAR2(255 BYTE),
+  "2018"                VARCHAR2(255 BYTE),
+  "2019"                VARCHAR2(255 BYTE)
```
Comment (duselmann) [inline liquibase_files/changelog1DailyTables.sql]: so my memory is correct that we did not do a 2018 refresh last year.

File: liquibase_files/changelog2_DailyYear.sql [hunk 2]
```diff
@@ -22,7 +22,7 @@ grant select on daily_year to seduser;
 -- insert into daily_year select SITE_NO,1905 from daily_years yrs where yrs."1905" is not null and yrs."1905" >0;
 -- insert into daily_year select SITE_NO,1906 from daily_years yrs where yrs."1906" is not null and yrs."1906" >0;
 -- insert into daily_year select SITE_NO,1907 from daily_years yrs where yrs."1907" is not null and yrs."1907" >0;
-insert into daily_year select SITE_NO,1908 from daily_years yrs where yrs."1908" is not null and yrs."1908" >0;
+--insert into daily_year select SITE_NO,1908 from daily_years yrs where yrs."1908" is not null and yrs."1908" >0;
```
Comment (duselmann) [inline liquibase_files/changelog2_DailyYear.sql]: Please remind me why we drop off years as we go.

**Comment by duselmann** [comment]
If this project is continued in the future then we have good patterns here to attempt a more automated refresh.

File: liquibase_files/changelog1DailyTables.sql (hunk 1 above)
Comment (supercasey) [reply liquibase_files/changelog1DailyTables.sql]: That is correct.

File: liquibase_files/changelog2_DailyYear.sql (hunk 2 above)
Comment (supercasey) [reply liquibase_files/changelog2_DailyYear.sql]: I don't intentionally, I repulled this data from NWIS, maybe these data were removed from the database.

**Pull request merged**